pkg/deployment/deployment.go

```diff
@@ -76,7 +76,10 @@ func Deploy(
 		readyWaitingMessage = fmt.Sprintf(dataplanev1beta1.ServiceReadyWaitingMessage, deployName)
 		readyMessage = fmt.Sprintf(dataplanev1beta1.ServiceReadyMessage, deployName)
 		readyErrorMessage = dataplanev1beta1.ServiceErrorMessage
-		aeeSpec.OpenStackAnsibleEERunnerImage = foundService.Spec.OpenStackAnsibleEERunnerImage
+		if foundService.Spec.OpenStackAnsibleEERunnerImage != "" {
+			aeeSpec.OpenStackAnsibleEERunnerImage = foundService.Spec.OpenStackAnsibleEERunnerImage
+		}
+
```

Comment:
based on this logic the behavior is that the image value on a service overrides what is set on the role.

Reply:
I believe this as already addressed.